src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -56,6 +56,30 @@ object VEP {
     (Locus(a(0), a(1).toInt), a(3) +: a(4).split(","))
   }
 
+  def getCSQHeaderDefinition(cmd: Array[String], confEnv: Map[String, String]): Option[String] = {
+    val csqHeaderRegex = "ID=CSQ[^>]+Description=\"([^\"]+)".r
+    val pb = new ProcessBuilder(cmd.toList.asJava)
+    val env = pb.environment()
+    confEnv.foreach { case (key, value) => env.put(key, value) }
+    
+    val (jt, proc) = List((Locus("1", 13372), IndexedSeq("G", "C"))).iterator.pipe(pb,
```

Comment:
Is this a trick to annotation one variant so as just to extract the header without doing a lot of work?